Doc/howto/logging.rst

```diff
@@ -472,7 +472,7 @@ hierarchical structures.  Multiple calls to :func:`getLogger` with the same name
 will return a reference to the same logger object.  Loggers that are further
 down in the hierarchical list are children of loggers higher up in the list.
 For example, given a logger with a name of ``foo``, loggers with names of
-``foo.bar``, ``foo.bar.baz``, and ``foo.bam`` are all descendants of ``foo``.
+``foo.bar``, ``foo.bar.baz``, and ``foo.bar`` are all descendants of ``foo``.
```

Comment:
I am not sure if it's a typo. I guess the statement is trying to illustrate that "foo.bam" is a descendant of "foo" like "foo.bar"

```
./python.exe
Python 3.9.0a0 (heads/master:042821ae3c, Jun 28 2019, 22:45:25)
[Clang 7.0.2 (clang-700.1.81)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import logging
>>> logging.getLogger("foo").parent
<RootLogger root (WARNING)>
>>> logging.getLogger("foo.bar").parent
<Logger foo (WARNING)>
>>> logging.getLogger("foo.bam").parent
<Logger foo (WARNING)>
>>> logging.getLogger("foo.bar.spam").parent
<Logger foo.bar (WARNING)>
```